src/EncompassRest.Tests/ContactNoteTests.cs

```diff
@@ -0,0 +1,48 @@
+﻿using EncompassRest.Contacts;
+using EncompassRest.Utilities;
+using Microsoft.VisualStudio.TestTools.UnitTesting;
+using System.Threading.Tasks;
+
+namespace EncompassRest.Tests
+{
+    [TestClass]
+    public class ContactNoteTests : TestBaseClass
+    {
+        [TestMethod]
+        public void ContactNote_Serialization()
+        {
+            var contactNote = new ContactNote { NoteId = "5" };
+            Assert.AreEqual(contactNote.NoteIdInt, 5);
+            Assert.AreEqual(@"{""noteId"":5}", contactNote.ToJson());
+            contactNote.Dirty = false;
+            Assert.AreEqual("{}", contactNote.ToJson());
+        }
+
+        [TestMethod]
+        public async Task ContactNote_CreateRetrieveAndDelete()
+        {
+            //create borrower contact to test notes
+            var client = await GetTestClientAsync();
+            var borrowerContact = new BorrowerContact();
+            borrowerContact.FirstName = "test";
+            borrowerContact.PersonalEmail = "me@me.com";
+            var contactId = await client.BorrowerContacts.CreateContactAsync(borrowerContact);
+            borrowerContact.Initialize(client);
```

Comment:
We shouldn't need to call `Initialize`, this should be done by `CreateContactAsync`.